main.tf

```diff
@@ -19,6 +19,9 @@ resource "metakube_cluster" "this" {
           id     = var.openstack_application_credential_id
           secret = var.openstack_application_credential_secret
         }
+        network     = var.openstack_network_config.network_id != null && var.openstack_network_config.subnet_id != null ? var.openstack_network_config.network_id : null
+        subnet_id   = var.openstack_network_config.network_id != null && var.openstack_network_config.subnet_id != null ? var.openstack_network_config.subnet_id : null
+        subnet_cidr = (var.openstack_network_config.network_id == null && var.openstack_network_config.subnet_id == null) && var.openstack_network_config.cidr != null ? var.openstack_network_config.cidr : null
```

Comment:
Can you align the names of the vars?

```
network != network_id
subnet_id = subnet_id
subnet_cidr = cidr 
```

That is confusing, one is named the same here and in `variables.tf`, the other two are not.